NineChronicles.Headless/GraphTypes/StateQuery.cs

```diff
@@ -699,6 +701,22 @@ public StateQuery()
                     return result;
                 }
             );
+
+            Field<StringGraphType>(
+                name: "cachedSheet",
+                arguments: new QueryArguments(
+                    new QueryArgument<StringGraphType>
```

Comment:
(It is not critical because we expect this API will be used only for internal operation)

```suggestion
                    new QueryArgument<NonNullGraphType<StringGraphType>>
```